### PR TITLE
Postgres container

### PR DESCRIPTION
**Pull request by vijay-veeranki-moj**
This is to update the User guide to add a walkthrough of using a postgres container as a test/dev DB.

### CODE REVIEW

File: source/documentation/other-topics/Setup-postgres-container.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    kubectl port-forward --namespace <env-name> svc/<YourName>-postgresql 5432:5432 &
+    PGPASSWORD="$POSTGRES_PASSWORD" psql --host 127.0.0.1 -U postgres -d my-database -p 5432
+
+
+Congratulations on getting this far. If all went well your postgresql pod is now deployed and you could connect to your database from outside the cluster.
+
+
+[env-create]: tasks.html#creating-a-cloud-platform-environment
+[auth-to-cluster]: tasks.html#authentication
```
Comment (digitalronin) [inline source/documentation/other-topics/Setup-postgres-container.md]: There is a mixture of absolute and relative URLs here, for links to other parts of the user guide. Please replace all the absolute URLs with relative ones.

File: source/documentation/other-topics/Setup-postgres-container.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ - You have [created an environment for your application][env-create]
+ - You have installed [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) on your local machine.
+ - You have [Authenticated][auth-to-cluster] to the cloud-platform-live-1 cluster.
+ - You have configured [Helm and Tiller][using-helm]. 
+
+##### Set up
+First we need to clone our reference application and change directory:
+
+    $ git clone https://github.com/ministryofjustice/cloud-platform-reference-app.git
+    $ cd cloud-platform-reference-app/helm_deploy/django-app/charts/postgresql
```
Comment (digitalronin) [inline source/documentation/other-topics/Setup-postgres-container.md]: This doesn't work. There is no "postgresql" directory on that path, only a `postgresql-4.0.1.tgz` file.

Also, is it really necessary to clone the entire reference app repo. if you only want the helm chart? Isn't there a way to just fetch the part of that repo the user actually needs? I think that would be less confusing.

Finally, do they really need to use the reference app. repo at all? If all they want is a standard postgres container, isn't it easier to use a "stable" helm chart (if there is one), or create a deployment using the bitnami postgresql container?

File: source/documentation/other-topics/Setup-postgres-container.md (hunk 1 above)
Comment (vijay-veeranki-moj) [reply source/documentation/other-topics/Setup-postgres-container.md]: Thanks @digitalronin updated the URL's to relative ones.

File: source/documentation/other-topics/Setup-postgres-container.md (hunk 2 above)
Comment (vijay-veeranki-moj) [reply source/documentation/other-topics/Setup-postgres-container.md]: Thanks @digitalronin, Sorry there is a PR for it, once it get merged we will have a "postgresql" directory
https://github.com/ministryofjustice/cloud-platform-reference-app/pull/76

Comment (vijay-veeranki-moj) [reply source/documentation/other-topics/Setup-postgres-container.md]: As now the above changes got merged, updated the user-guide considering the review comments.

**Pull request merged**